service/hook/github/github.go

```diff
@@ -251,6 +244,7 @@ func (hp HookProvider) TransformRequest(r *http.Request) hookCommon.TransformRes
 	}
 }
 
+// returns the repository clone URL depending on the publicity of the project
```

Comment:
what's `SSHURL` and what's `CloneURL`? is `CloneURL` always the public / HTTPS url and SSH the `git@..` ssh/private URL?